backend/battle/tests/test_view.py

```diff
@@ -74,3 +80,343 @@ def test_verify_if_not_returns_winner_missing_pokemons_teams(self):
 
         with self.assertRaises(IndexError):
             get_winner_for(self.team_creator, self.team_opponent)
+
+
+class ListBattlesTest(TestCaseUtils):
+    def setUp(self):
+        super().setUp()
+        self.battle = baker.make('battle.Battle', creator=self.user)
+        self.battle_from_user = Battle.objects.filter(creator=self.user)
```

Comment:
Please avoid doing queries that will be used on asserts on the setup, add them explicitly on the `self.assertCountEqual`

Reply:
Got it! :)